hack/generate/samples/internal/go/memcached-with-webhooks/e2e_test_code.go

```diff
@@ -288,15 +288,15 @@ var _ = Describe("memcached", func() {
 
 			By("validating that pod(s) status.phase=Running")
 			getMemcachedPodStatus := func() error {
-				cmd = exec.Command("kubectl", "get",
-					"pods", "-l", "app=memcached", "-o", "jsonpath={.items[*].status}",
-					"-n", namespace,
+				cmd =  exec.Command("kubectl", "get", "-l",
+					"app.kubernetes.io/name=memcached",
+					"-o", "jsonpath={.items[*].status}",
 				)
 				status, err := utils.Run(cmd)
 				fmt.Println(string(status))
 				ExpectWithOffset(2, err).NotTo(HaveOccurred())
-				if !strings.Contains(string(status), "\"phase\":\"Running\"") {
-					return fmt.Errorf("memcached pod in %s status", status)
+				if !strings.Contains(status, "\"phase\":\"Running\"") {
+					return err
```

Comment:
It fails we need to use string(status) to convert the byte:
See: 

```
Error: test/e2e/e2e_test.go:172:26: cannot use status (variable of type []byte) as type string in argument to strings.Contains
FAIL	github.com/example/memcached-operator/test/e2e [build failed]
FAIL
make[1]: *** [Makefile:110: test-e2e] Error 2
make[1]: Leaving directory '/home/runner/work/operator-sdk/operator-sdk/testdata/go/v3/memcached-operator'
make: *** [Makefile:193: test-e2e-sample-go] Error 2
```

@NikhilSharmaWe here we can revert all changes